### PR TITLE
Improve demo

### PR DESCRIPTION
**Pull request by blasten**
screen shot 2015-05-29 at 11 27 26 am

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -21,32 +21,57 @@
 
   <link rel="import" href="../iron-input.html">
 
-  <link rel="stylesheet" href="../../paper-styles/demo.css">
+  <link href="../../paper-styles/demo-pages.html" rel="import">
+    <link href="../../paper-button/paper-button.html" rel="import">
```
Comment (notwaldorf) [inline demo/index.html]: I don't think `paper-button` is in `bower.json`

File: demo/index.html [hunk 2]
```diff
@@ -21,32 +21,57 @@
...
+    input, button {
+      font-size: 20px;
+      padding: 0.2em;
+    }
+  </style>
+</head>
+<body>
+  <div class="vertical-section vertical-section-container centered">
+    <template is="dom-bind">
+      <section>
```
Comment (notwaldorf) [inline demo/index.html]: do you need the `<section>`? I think the `<vertical-section>` is probably enough

**Comment by notwaldorf** [comment]
lgtm % comments

**Comment by blasten** [comment]
It should be all good now :)

**Pull request merged**